src/dbnode/digest/writer.go

```diff
@@ -109,3 +112,37 @@ func (w *fdWithDigestContentsWriter) WriteDigests(digests ...uint32) error {
 	}
 	return nil
 }
+
+func CloseAll(writers ...FdWithDigestWriter) error {
+	multiErr := xerrors.NewMultiError()
+	for _, writer := range writers {
+		if err := writer.Close(); err != nil {
+			multiErr = multiErr.Add(err)
+		}
+	}
+	return multiErr.FinalError()
+}
+
+func SyncAll(writers ...FdWithDigestWriter) error {
+	var (
+		multiErr = xerrors.NewMultiError()
+		errMut   = sync.Mutex{}
```

Comment:
nit: typically just `mu sync.Mutex` (no need to scope to errMut since no other mutexes here)

Reply:
👍